packages/experimental/Button/src/Button.types.ts

```diff
@@ -111,16 +111,62 @@ export interface ButtonProps extends Omit<IWithPressableOptions<ViewProps>, 'onP
   fluid?: boolean;
 }
 
+export interface ButtonMacProps {
+  /*
+   * Name of the icon to show on the Button.
+   */
+  image?: Image;
```

Comment:
I learned from working on `Avatar` that passing an `image` is actually not quite simple, and it may be easier to pass an `ImageURISource` that the component internally calls `Image.resolveAssetSource(uri: ...)` on so that RCTConvert can properly convert it to a UIImage / NSImage

Reply:
Updated image type to ImageURISource.